apps/web/app/(preview)/layout.tsx

```diff
@@ -10,7 +12,11 @@ export default function RootLayout({ children }: { children: React.ReactNode })
   return (
     <html lang="en">
       <ZoomListener />
-      <body>{children}</body>
+
+      <body>
+        <Toaster position="top-right" closeButton={true} />
+        {children}
+      </body>
```

Comment:
This is not final. What would be the best way to implement the VIBES Toaster for all examples for a specific vibe? Or do we need to add this on a per example basis? Open to suggestions.

Reply:
We'll probably want to have a preview root layout for each VIBE. Maybe we can accomplish that using [parallel routes](https://nextjs.org/docs/app/building-your-application/routing/parallel-routes)?

I'm ok cutting this corner for now but let's add a comment in this file to make it clear that this is hardcoded to the Soul VIBE's Toaster primitive and we would want to have each VIBE define its own layout.

Another idea is to export the layout from within each VIBE and then build a map of `lazy` imports and then use the VIBE slug to get the appropriate layout. We'd have to render it further down the routes so we can access the parameter, though.